src/application.js

```diff
@@ -5,9 +5,33 @@
 Marionette.Application = Marionette.Object.extend({
   cidPrefix: 'mna',
 
+  rootEl: 'body',
+
+  constructor: function(options) {
+    options = options || {};
+
+    this._initRootRegion(options);
+
+    Marionette.Object.prototype.constructor.apply(this, arguments);
+  },
+
+  _initRootRegion: function(options) {
+    var rootEl = options.rootEl || this.rootEl;
+
+    rootEl = Marionette._getValue(rootEl, this, options);
+
+    // rootEl is optional
+    if (rootEl) {
+
+      var RootRegion = Marionette.Region.extend({
+        el: rootEl
+      });
+
+      this.rootEl = new RootRegion();
```

Comment:
Yeah I was following this:
```
var myApp = new Application({
  rootEl: "#my-app"
}); 

myApp.on("start", () =>
   this.rootEl.show(new FooLayout)
)
```
I agree with you about the separation, but what we set here will be what you call show on.. so I suppose two options..
we could store the original `rootEl` selector string on `this._rootEl` kind of like uiBindings..  or we could make the region name different than the selector, but I'd suggest something without the leading _?  maybe just root? so:  `this.root.show(new FooLayout)`